l10n_it_mis_reports/i18n/it.po

```diff
@@ -0,0 +1,959 @@
+msgid "E) Accruals and deferrals"
+msgstr "E) Ratei e risconti"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_pl_e
+msgid "E) Extraordinary gains and losses"
+msgstr "E) Proventi e oneri straordinari"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsa_bi
+msgid "I) Intangible fixed assets"
+msgstr "I - Immobilizzazioni immateriali"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsa_ci
+msgid "I) Inventory"
+msgstr "I - Rimanenze"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsl_ai
+msgid "I) Share capital"
+msgstr "I. Capitale"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsl_aii
+msgid "II) Paid-in capital in excess of par"
+msgstr "II. Riserve da sovrapprezzo delle azioni"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsa_cii
+msgid "II) Receivables"
+msgstr "II - Crediti"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsa_bii
+msgid "II) Tangible Fixed Assets"
+msgstr "II - Immobilizzazioni materiali"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsa_ciii
+msgid "III) Current financial assets"
+msgstr "III - Attività finanziarie che non costituiscono immobilizzazioni"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsa_biii
+msgid "III) Financial fixed assets"
+msgstr "III) Immobilizzazioni finanziarie"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsl_aiii
+msgid "III) Revaluation reserves"
+msgstr "III. Riserve di rivalutazione"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_plx
+msgid "INCOME STATEMENT"
+msgstr "CONTO ECONOMICO"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsa_civ
+msgid "IV) Cash and cash equivalents"
+msgstr "IV - Disponibilità liquide"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsl_aiv
+msgid "IV) Legal reserve"
+msgstr "IV. Riserva legale"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report.kpi,description:l10n_it_mis_reports.mis_report_bsl_aix
+msgid "IX) Current earnings (losses)"
+msgstr "IX. Utile (perdita) dell'esercizio"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report,name:l10n_it_mis_reports.mis_report_bs
+msgid "Italian Balance Sheet (Full Model)"
+msgstr "Bilancio CEE Stato Patrimoniale"
+
+#. module: l10n_it_mis_reports
+#: model:mis.report,name:l10n_it_mis_reports.mis_report_pl
+msgid "Italian P&L (Full Model)"
+msgstr "Bilancio CEE Conto Economico"
```

Comment:
La traduzione è incoerente. Eventualmente da modificare il termine originale.

In generale i termini inglesi delle voci di bilancio civilistico li hai trovati sulla normativa?

Reply:
le traduzioni le ho prese da : 
https://it.wikiversity.org/wiki/Conto_economico
https://it.wikiversity.org/wiki/Stato_patrimoniale